From 06cf9eba3b4303a9f297808935ab8f5f710103c7 Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Tue, 8 Aug 2017 08:37:03 -0700
Subject: [PATCH] joyent/sdc-imgapi#14 Fix missing .com for github repos in
 README.md Reviewed by: Todd Whiteman <todd.whiteman@joyent.com> Approved by:
 Todd Whiteman <todd.whiteman@joyent.com>

---
 README.md | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/README.md b/README.md
index 443379c..1462d6d 100644
--- a/README.md
+++ b/README.md
@@ -73,8 +73,8 @@ management in SmartDataCenter and SmartOS.
 
 | Repo | Description |
 | ---- | ----------- |
-| https://github/joyent/sdc-imgapi.git | The IMGAPI server. This repository. |
-| https://github/joyent/node-sdc-clients.git | Includes imgapi.js node client library for using IMGAPI. |
-| https://github/joyent/node-imgmanifest.git | Defines the SDC/SmartOS Image manifest spec and support for validation and upgrade of image manifests. |
-| https://github/joyent/sdc-imgapi-cli.git | Includes the `*-imgadm` tools for common IMGAPI servers, e.g. `joyent-imgadm`, `sdc-imgadm`, `updates-imgadm` and a framework for tools for other IMGAPI servers. |
-| https://github/joyent/smartos-live.git | Holds the SmartOS `imgadm` tool (in src/img). This is the tool used on any SmartOS global zone to install image datasets into the zpool to enable provisioning VMs with that image. |
+| https://github.com/joyent/sdc-imgapi.git | The IMGAPI server. This repository. |
+| https://github.com/joyent/node-sdc-clients.git | Includes imgapi.js node client library for using IMGAPI. |
+| https://github.com/joyent/node-imgmanifest.git | Defines the SDC/SmartOS Image manifest spec and support for validation and upgrade of image manifests. |
+| https://github.com/joyent/sdc-imgapi-cli.git | Includes the `*-imgadm` tools for common IMGAPI servers, e.g. `joyent-imgadm`, `sdc-imgadm`, `updates-imgadm` and a framework for tools for other IMGAPI servers. |
+| https://github.com/joyent/smartos-live.git | Holds the SmartOS `imgadm` tool (in src/img). This is the tool used on any SmartOS global zone to install image datasets into the zpool to enable provisioning VMs with that image. |
-- 
2.21.0

